docs/components/Chips/Chips.stories.mdx

```diff
@@ -42,39 +31,6 @@ Unlike
 selected single-select Chip can be de-selected by the user, leaving all
```

Comment:
The above link for radio is broken and should be `[Radio](../?path=/docs/components-forms-and-inputs-radiogroup-docs--page)`

Reply:
Nice catch!